tests/system/test_dbapi.py

```diff
@@ -152,7 +154,7 @@ def test_commit_exception(self):
         try:
             self._conn.commit()
         except Exception:
-            pass
+            self._conn.database._pool._sessions.get()
```

Comment:
If I remember correctly, this change is needed, because in this case no session has been checked out of the pool. Is this something that could also change for user code? In other words: Could it be that users have code in their applications that trigger this same thing, and that previously worked without an error, and that would now throw an error?

Reply:
Not completely sure but seems like something has changed in the session logic at python client or Span FE as this test was working before but has started failing (irrespective of changes in this PR). 
So in this test we are deleting the session at SpanFE so that commit would fail and we catch the exception. We then try to test that a new transaction works on the same connection but seems like now the new transaction is trying to checkout the same session which was deleted resulting in `Session not found` exception. So to fix this I am removing the session from the session pool at client by getting the first session from the queue

Reply:
OK, would you mind adding a comment to the code here to explain why we are doing this.